index.html

```diff
@@ -0,0 +1,420 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link href="assets/css/style.css" rel="stylesheet" type="text/css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css"
+          integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU"
+          crossorigin="anonymous">
+    <title>Kiryl Poyu curriculum vitae</title>
+</head>
+<body>
+<main>
+    <header id="start-anchor">
```

Comment:
maybe it is better to use div as a wrapper and such structure as header + main + footer inside?

Reply:
fix it